iXBRLViewerPlugin/ui.py

```diff
@@ -107,3 +178,39 @@ def filename(self):
 
     def zipViewerOutput(self):
         return self._zipViewerOutput.get()
+
+
+class SettingsDialog(ViewerDialog):
+    """
+    Dialog for saving default viewer settings.
+    """
+    def __init__(self, cntlr):
+
+        self.cntlr = cntlr
+        self.parent = cntlr.parent
+
+        self._launchOnLoad = BooleanVar()
+        self._launchOnLoad.set(self.cntlr.config.setdefault(CONFIG_LAUNCH_ON_LOAD, True))
+
+        super(SettingsDialog, self).__init__(cntlr, "iXBRL Viewer Settings")
+
+    def addFields(self, frame: Frame, y: int) -> int:
+        y += 1
+        launchOnLoadCheckbutton = Checkbutton(frame, text="Launch on Load", variable=self._launchOnLoad, onvalue=True, offvalue=False)
+        launchOnLoadLabel = Label(frame, text="Launches an instance of the viewer in Stub Viewer Mode whenever a document is loaded.")
```

Comment:
```suggestion
        launchOnLoadLabel = Label(frame, text="Launches an instance of the viewer in your default browser  whenever an Inline XBRL report is loaded.")
```

I think that the fact that the local viewer uses stub viewer mode is incidental here.